calibration/PHSEND/CGINS-PHSEND-P0087__20190401.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0087,CC_ea434,17372.0,source_file PHSEN-D_SAMI-pH_SN_P0087_Validation_Files_2019-04-01.zip > SAMI_P0087_010419_tris_ValCert.pdf
+P0087,CC_ea578,94.1,
+P0087,CC_eb434,2284.0,
+P0087,CC_eb578,38676.0,
+P0087,CC_ind_off,0,
+P0087,CC_ind_slp,1,
+P0087,CC_psal,35,Constant
```

Comment:
Looks good. Again there are extra directory levels here, but maybe those aren't copied to Alfresco?

Reply:
Looks good.  Not worried about the extra folder levels.  If we know the file name we can find it in the folder.